src/main/java/ui/IdGenerator.java

```diff
@@ -0,0 +1,125 @@
+package ui;
+
+public final class IdGenerator {
+    private IdGenerator() {}
+
+    public static String getPanelCellId(int panelIndex, int issueId) {
+        return "panel" + panelIndex + "_" + issueId;
+    }
+
+    public static String getPanelFilterTextFieldId(int panelIndex) {
+        return "panel" + panelIndex + "_filterTextField";
+    }
+
+    public static String getPanelRenameTextFieldId(int panelIndex) {
+        return "panel" + panelIndex + "_renameTextField";
+    }
+
+    public static String getRepositorySelectorId() {
+        return "repositorySelector";
+    }
+
+    public static String getPanelNameAreaId(int panelIndex) {
+        return "panel" + panelIndex + "_nameText";
+    }
+
+    public static String getPanelId(int panelIndex) {
+        return "panel" + panelIndex;
+    }
+
+    public static String getLabelPickerTextFieldId() {
+        return "queryField";
+    }
+
+    public static String getPanelCloseButtonId(int panelIndex) {
+        return "panel" + panelIndex + "_closeButton";
+    }
+
+    public static String getPanelRenameButtonId(int panelIndex) {
+        return "panel" + panelIndex + "_renameButton";
+    }
+
+    public static String getOcticonButtonId(int panelIndex, String octiconCssName) {
+        return "panel" + panelIndex + "_" + octiconCssName;
+    }
+
+    public static String getLoginDialogOwnerFieldId() {
+        return "repoOwnerField";
+    }
+
+    public static String getBoardNameInputFieldId() {
+        return "boardnameinput";
+    }
+
+    public static String getBoardNameSaveButtonId() {
+        return "boardsavebutton";
+    }
+
+    public static String getApiBoxId() {
+        return "apiBox";
+    }
+
+    public static String getPanelCellIdForTest(int panelIndex, int issueId) {
```

Comment:
We may not need the `ForTest` suffix here. These may be used in other contexts as well.